CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+## How to contribute code
```

Comment:
Should we include some discussion of filing an issue somewhere here? Could be a good place to capture any expectations we have (like a colab reproduction once we are fully packaged on pypi).

Reply:
Done - thanks!